### ISSUE TITLE
file_server not handling BrokenPipe

### ISSUE THREAD

**Issue by t829702** [denoland/deno#5624]
is this worth to fix? and is there another full featured file_server in the community out of std?

easy to reproduce this when serving a large file to browser, and click stop loading

```console
[2020-05-19 12:00:25] "GET /output.json HTTP/1.1" 200
error: Uncaught BrokenPipe: Broken pipe (os error 32)
    at unwrapResponse ($deno$/ops/dispatch_minimal.ts:63:11)
    at Object.sendAsyncMinimal ($deno$/ops/dispatch_minimal.ts:106:10)
    at async Object.write ($deno$/ops/io.ts:65:18)
    at async BufWriter.flush (https://deno.land/std/io/bufio.ts:475:25)
    at async BufWriter.write (https://deno.land/std/io/bufio.ts:514:9)
    at async Object.copy ($deno$/io.ts:70:21)
    at async writeResponse (https://deno.land/std/http/_io.ts:265:15)
    at async ServerRequest.respond (https://deno.land/std/http/server.ts:97:7)
```

**Comment by jimvandervoort** [comment]
I would like to work on this :)

**Comment by bartlomieju** [comment]
Fixed in #6285
